examples/Demo/FluentUI.Demo.Client/Documentation/GetStarted/Examples/McpCapabilities.razor

```diff
@@ -0,0 +1,131 @@
+                                    @if (param.Required)
+                                    {
+                                        <span>*</span>
+                                    }
+                                </span>
+                            }
+                        </FluentStack>
+                    }
+                    else
+                    {
+                        <span class="text-muted">None</span>
+                    }
+                </TemplateColumn>
+            </FluentDataGrid>
+        </FluentTab>
+
+        <FluentTab Id="prompts" Label="@($"Prompts ({_summary.Prompts.Count})")">
+            <FluentDataGrid TGridItem="McpPromptInfo" Items="@_summary.Prompts.AsQueryable()" GridTemplateColumns="1fr 2fr 1fr">
+                <PropertyColumn Property="@(p => p.Name)" Title="Name" />
+                <PropertyColumn Property="@(p => p.Description)" Title="Description" />
+                <TemplateColumn Title="Parameters">
+                    @if (context.Parameters.Count > 0)
+                    {
+                        <FluentStack Orientation="Orientation.Vertical" VerticalGap="0">
+                            @foreach (var param in context.Parameters)
+                            {
+                                <span>
+                                    <code>@param.Name</code>
+                                    <span class="text-muted">(@param.Type)</span>
+                                    @if (param.Required)
+                                    {
+                                        <span>*</span>
+                                    }
+                                </span>
+                            }
+                        </FluentStack>
+                    }
+                    else
+                    {
+                        <span class="text-muted">None</span>
+                    }
+                </TemplateColumn>
+            </FluentDataGrid>
+        </FluentTab>
+
+        <FluentTab Id="resources" Label="@($"Resources ({_summary.Resources.Count})")">
+            <FluentDataGrid TGridItem="McpResourceInfo" Items="@_summary.Resources.AsQueryable()" GridTemplateColumns="1fr 1fr 2fr 0.5fr">
+                <PropertyColumn Property="@(r => r.Uri)" Title="URI" />
+                <PropertyColumn Property="@(r => r.Title)" Title="Title" />
+                <PropertyColumn Property="@(r => r.Description)" Title="Description" />
+                <TemplateColumn Title="Type">
+                    @if (context.IsTemplate)
+                    {
+                        <span class="text-muted">Template</span>
+                    }
+                    else
+                    {
+                        <span class="text-muted">Static</span>
+                    }
+                </TemplateColumn>
+            </FluentDataGrid>
+        </FluentTab>
+    </FluentTabs>
+}
+
+@code {
```

Comment:
If we don't display the source code in the `.md` file, can you move this code to a `.cs` file ?